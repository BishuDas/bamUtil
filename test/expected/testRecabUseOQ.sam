@HD	VN:1.0	SO:unsorted
@SQ	SN:1	LN:249250621
@SQ	SN:2	LN:243199373
@RG	ID:rg1	LB:lb1	SM:sm1
1	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
2	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
3	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
4	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
5	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
6	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
7	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
8	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
9	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
10	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
11	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
12	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
13	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
14	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
15	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
16	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
17	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
18	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
19	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
20	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
21	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
22	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
23	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
24	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
25	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
26	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
27	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
28	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
29	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
30	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
31	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
32	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
33	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
34	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
35	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
36	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
37	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
38	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
39	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
40	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
41	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
42	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
43	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
44	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
45	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
46	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
47	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
48	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
49	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
50	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
51	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
52	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
53	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
54	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
55	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
56	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
57	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
58	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
59	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
60	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
61	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
62	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
63	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
64	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
65	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
66	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
67	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
68	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
69	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
70	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
71	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
72	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
73	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
74	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
75	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
76	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
77	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
78	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
79	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
80	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
81	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
82	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
83	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
84	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
85	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
86	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
87	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
88	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
89	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
90	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
91	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
92	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
93	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
94	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
95	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
96	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
97	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
98	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
99	0	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
101	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
102	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
103	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
104	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
105	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
106	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
107	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
108	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
109	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
110	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
111	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
112	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
113	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
114	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
115	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
116	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
117	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
118	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
119	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
120	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
121	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
122	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
123	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
124	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
125	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
126	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
127	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
128	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
129	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
130	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
131	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
132	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
133	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
134	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
135	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
136	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
137	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
138	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
139	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
140	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
141	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
142	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
143	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
144	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
145	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
146	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
147	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
148	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
149	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
150	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
151	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
152	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
153	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
154	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
155	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
156	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
157	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
158	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
159	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
160	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
161	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
162	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
163	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
164	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
165	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
166	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
167	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
168	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
169	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
170	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
171	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
172	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
173	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
174	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
175	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
176	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
177	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
178	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
179	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
180	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
181	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
182	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
183	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
184	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
185	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
186	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
187	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
188	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
189	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
190	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
191	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
192	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
193	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
194	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
195	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
196	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
197	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
198	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
199	131	1	10021	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
100	0	1	10022	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
200	131	1	10022	10	5M	=	10	0	ACCCT	!5522	OQ:Z:!<<<<	RG:Z:rg1
300	131	1	10400	10	5M	=	10	0	TTTTC	!$!!$	OQ:Z:LMNOP	RG:Z:rg1
