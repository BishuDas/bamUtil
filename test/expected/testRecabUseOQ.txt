Model
Effect Variance SE
-2.034487 1308.223209 36.169368 
0.697072 4.971137 2.229605 
-0.742840 2.192206 1.480610 
0.000006 1.009437 1.004707 
-9.211267 458.795724 21.419517 
-0.076566 1.069533 1.034183 
